release-notes.md

```diff
@@ -1,3 +1,31 @@
+# Release 1.9.9
+
+## What's New
+* none
```

Comment:
call out the removal of the 'old' cert and that from here forward 'old' clients cannot auto-update. they must uninstall/reinstall